<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacherCreateQuiz</class>
 <widget class="QMainWindow" name="teacherCreateQuiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>830</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Teacher Create Quiz</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Desktop/javaicon.png</normaloff>../../../../../Desktop/javaicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="overallLayout">
      <item>
       <widget class="QLabel" name="bg">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>Image/gg2.JPG</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="menu">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="homeBtn">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border: None;
}

QPushButton:hover
{
	color: rgb(47,79,79);
	font-size: 34px;
}

</string>
          </property>
          <property name="text">
           <string>Home</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="viewquizBtn">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border: None;
}

QPushButton:hover
{
	color: rgb(47,79,79);
	font-size: 34px;
}

</string>
          </property>
          <property name="text">
           <string>View Quiz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createquizBtn">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border: None;
}

QPushButton:hover
{
	color: rgb(47,79,79);
	font-size: 34px;
}

</string>
          </property>
          <property name="text">
           <string>Create Quiz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Content">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QLabel" name="title">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Teacher Create Quiz</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="quiztitleLayout" stretch="5,1">
        <item>
         <widget class="QLineEdit" name="createquiztitletxtbox">
          <property name="maximumSize">
           <size>
            <width>857</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Quiz Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createquiztitleBtn">
          <property name="text">
           <string>Create Quiz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="menu2">
        <item>
         <widget class="QPushButton" name="mcqBtn">
          <property name="text">
           <string>Add MCQ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tfBtn">
          <property name="text">
           <string>Add T/ F</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saBtn">
          <property name="text">
           <string>Add Short Answer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="qnLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="Content2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="mcqWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="mcqLayout">
            <item>
             <widget class="QLineEdit" name="mcqqnTextbox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Input Question</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="mcqGrpBox">
              <property name="title">
               <string>Select Correct Answer</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="combo1">
                 <item>
                  <widget class="QCheckBox" name="cb1">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="cbtxt1">
                   <property name="placeholderText">
                    <string>Option 1</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="combo4">
                 <item>
                  <widget class="QCheckBox" name="cb4">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="cbtxt4">
                   <property name="placeholderText">
                    <string>Option 4</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="combo3">
                 <item>
                  <widget class="QCheckBox" name="cb3">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="cbtxt3">
                   <property name="placeholderText">
                    <string>Option 3</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="combo2">
                 <item>
                  <widget class="QCheckBox" name="cb2">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="cbtxt2">
                   <property name="placeholderText">
                    <string>Option 2</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QPushButton" name="saveBtn_2">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="tfWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="tfLayout">
            <item>
             <widget class="QLineEdit" name="tfqnTextbox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1078</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Input Question</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="radioGrpBox">
              <property name="title">
               <string>Select Correct Answer</string>
              </property>
              <widget class="QRadioButton" name="tradioButton">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>30</y>
                 <width>119</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>True</string>
               </property>
              </widget>
              <widget class="QRadioButton" name="fradioBtn">
               <property name="geometry">
                <rect>
                 <x>170</x>
                 <y>30</y>
                 <width>119</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>False</string>
               </property>
              </widget>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QPushButton" name="saveBtn">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="saWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="saLayout">
            <item>
             <widget class="QLineEdit" name="saTextbox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Input Question</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="saansTextbox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Input Answer</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUsers">
    <property name="title">
     <string>Users</string>
    </property>
    <addaction name="actionLogout"/>
    <addaction name="actionLogout_2"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionClose_Alt_F4"/>
   </widget>
   <addaction name="menuUsers"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionClose_Alt_F4">
   <property name="text">
    <string>Close (Alt + F4)</string>
   </property>
  </action>
  <action name="actionLogout_2">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
